### ISSUE TITLE
Weird noise appears when saving audioclips

### ISSUE THREAD

**Issue by Vijayabhaskar96** [Zulko/moviepy#1174]
### Expected Behavior
When I do this:
```
audio_file = AudioFileClip("source.wav")
audio_file.write_audiofile("noisy.mp3")
```
I expect both audios to sound the same, but the saved file `noisy.mp3`  has a weird added noise at the end.
You can see the results here:
Source file: https://bayfiles.com/L5b3tfv8oe/source_wav
Noisy File produced: https://bayfiles.com/3fc1t1vao6/noisy_mp3

### Specifications

Tested on Python 3.7 on Google Colab with Latest git dev version of the moviepy
`!pip install git+https://github.com/Zulko/moviepy.git --upgrade` 
and on Windows 10 with moviepy version 1.0.0

**Comment by tburrows13** [comment]
Thanks for the report. I can confirm that it is reproducible in the current dev version of v2.0, but it is fixed in #1108, which will be merged before the release of v2.0.

**Comment by tburrows13** [comment]
Closed in #1108.
